packages/server/database/migrations/20200807100000-addCheckInMetrics.ts

```diff
@@ -0,0 +1,54 @@
+import {R} from 'rethinkdb-ts'
+import {MeetingTypeEnum} from 'parabol-client/types/graphql'
+
+export const up = async function(r: R) {
+  try {
+    const getThreadIds = (row) =>
+      r.args(
+        r
+          .table('AgendaItem')
+          .getAll(row('id'), {index: 'meetingId'})
+          .filter({isActive: true})('id')
+          .coerceTo('array') as any
+      )
+    await r
+      .table('NewMeeting')
+      .filter({meetingType: MeetingTypeEnum.action})
+      .filter((row) =>
+        row('endedAt')
+          .default(null)
+          .ne(null)
+      )
+      .update(
+        (row) => ({
+          commentCount: r
+            .table('Comment')
+            .getAll(getThreadIds(row), {index: 'threadId'})
```

Comment:
this migration doesn't work. you're passing in a comment, but on L10 you're using the `meetingId` as an index.
to test, make sure that on the timeline the agenda item count & the comment count return a non-zero for some of the completed meetings.

Reply:
Hmm I might be mistaken here but my understanding is that in the `.update` we're looping through each `NewMeeting` and passing a `meeting` into the `getThreadIds`, not a `comment`? That function returns the agenda items ids which we use to grab the comments.

I've updated the migration removing the active filter and how `agendaItemCount` was calculated and I'm now getting non-zero counts for completed meetings.